demos/vite-demo/src/main.tsx

```diff
@@ -0,0 +1,10 @@
+import { StrictMode } from 'react';
+import { createRoot } from 'react-dom/client';
+import App from './App.tsx';
+import './index.css';
+
+createRoot(document.getElementById('root')!).render(
+  <StrictMode>
```

Comment:
Also, isograph won't run in `StrictMode`.

Reply:
Fixed!